edit-client/src/client.rs

```diff
@@ -321,13 +322,18 @@ pub trait ClientImpl {
     where
         Self: Sized,
     {
-        self.send_client(&UserToFrontendCommand::Controls(Controls{
+        let mut controls_object = &UserToFrontendCommand::Controls(Controls{
```

Comment:
Here, I think you should make `let controls_object = Controls { ... }` and not wrap it in `UserToFrontendCommand::Controls` yet.